src/PrestaShopBundle/Resources/config/services/bundle/repository.yml

```diff
@@ -2,10 +2,6 @@ services:
     _defaults:
         public: true
 
-    prestashop.module_kernel.repository:
```

Comment:
BC breaks, we should keep the file, mark it as deprecated, and keep the service for compatibility reason.